packages/prisma/migrations/20230523102615_added_booker_layouts/migration.sql

```diff
@@ -0,0 +1,5 @@
+-- AlterTable
+ALTER TABLE "EventType" ADD COLUMN     "bookerLayouts" JSONB;
+
+-- AlterTable
+ALTER TABLE "users" ADD COLUMN     "defaultBookerLayouts" JSONB;
```

Comment:
Should we use users `metadata` here instead of adding a new column to our 90k plus users?

Reply:
I just mention this because we already have quiet a few bits of new settings for a specific user that lives in there. e.g. defaultConferncingApp lives in there feels like they are the same sort of thing

Reply:
Yes good question. I'm unsure what's the best approach. That's also why I added the question above. I'm okay with moving it. 

Let me know what you think @zomars or @emrysal

Reply:
Agreed with @sean-brydon let's use metadata for this.